query-service-impl/src/main/java/org/hypertrace/core/query/service/trino/converters/DefaultColumnRequestConverter.java

```diff
@@ -360,6 +360,9 @@ private String convertExpressionToString(
         trinoExecutionContext.addActualTableColumnName(columnName);
         ColumnRequestContext context = trinoExecutionContext.getColumnRequestContext();
         context.setColumnValueType(tableDefinition.getColumnType(logicalColumnName));
+        if (context.isBytesColumnType()) {
```

Comment:
Does this take care in filter too?

Reply:
Added bytes type support in filter. Also enabled tests.